apps/src/lab2/views/ExtraLinksModal.tsx

```diff
@@ -121,6 +131,28 @@ const ExtraLinksModal: React.FunctionComponent<ExtraLinksModalProps> = ({
     }
   };
 
+  const onResetAbuseScore = async () => {
+    try {
+      await HttpClient.post(
+        `/v3/channels/${channelId}/abuse/delete`,
+        '',
+        true,
+        {contentType: 'application/json;charset=UTF-8'}
+      );
+      setAbuseScore(0);
+    } catch (e) {
+      console.log('Error resetting abuse score to 0.');
```

Comment:
should we show an error message in the modal in this case?

Reply:
Updated! Here's a screencast video (manually throwing error when resetting abuse score to 0):

https://github.com/user-attachments/assets/d48f9877-7e1c-48f3-89d5-a442a93d1ef8

Reply:
Lmk what you think of latest commit - thanks! https://github.com/code-dot-org/code-dot-org/pull/62929/commits/a81356f259e41237e32ad010077f32791867ad54